recipes/repeatmodeler/meta.yaml

```diff
@@ -7,7 +7,7 @@ source:
   md5: c148b4b56bb3fbd4e9ba88ac8736baa3
 
 build:
-  number: 1
+  number: 2
   skip: True  #[osx]
 
 requirements:
```

Comment:
Did you miss adding the `nseg` dependency? Or did you want to add that in another PR?

Reply:
I'm not sure it has to be in this recipe because nseg is part of RepeatScout and is run only if RepeatScout is available.

Reply:
Ok, I agree it’s not needed here if that is the case. However, then why does the `NSEG_PRGM` configuration thing exist here?

Reply:
I would prefer to keep it like that for clarity